### ISSUE TITLE
Training deeplab on higher resolution images

### ISSUE THREAD

**Issue by AbdouMechraoui** [nv-tlabs/datasetGAN_release#39]
Hi!

Thanks once again for your responsiveness :)

May I ask about the procedure used to train `deeplab-v3` on 1024 images, I noticed the provided code in the repo samples `train_interpreter.py`, and trains `train_deeplab.py` on images with 512 x 512 resolution for face_34 task. 

When doing **the same for 1024 x 1024 images**, the cross_validation script results in this:
<img width="230" alt="trained_deeplab_1024_500" src="https://user-images.githubusercontent.com/52415829/144714148-25135813-f7b7-4f0e-aed5-fd6d130dd374.png">

Could you provide more information on the procedure for training deeplab on 1024 x 1024 images? How large was the training set that you used? The number of epochs, and batch size?  I ask the latter, as I also had some issues with OOM cuda error, when training on 32G NVIDIA Tesla V100.

**Comment by arieling** [comment]
Hi. In datasetGAN paper, we used 512 x 512 resolution. We didn't scale it up to 1024* 1024.

**Comment by AbdouMechraoui** [comment]
I am closing this, I was able to train deeplab using ~500 images/labels (1024, 1024), 30 epochs, and batch_size=4. Thanks anyways :)
